values.yaml

```diff
@@ -826,6 +826,7 @@ redis:
     jobserviceDatabaseIndex: "1"
     registryDatabaseIndex: "2"
     trivyAdapterIndex: "5"
+    username: ""
```

Comment:
\# Redis AUTH command was extended in Redis 6, it is possible to use it in the two-arguments AUTH <username> <password> form.

will it be better to add a comment for user to aware this is an optional config?